File: tests/fixtures/sample_code/Sample.cs
using System;
using System.Collections.Generic;

namespace Example.Services
{
    /// <summary>
    /// Sample C# service class for testing.
    /// </summary>
    public class CustomerService
    {
        private readonly List<string> _customers = new List<string>();

        public void RegisterCustomer(string name)
        {
            if (!string.IsNullOrEmpty(name))
            {
                _customers.Add(name);
            }
        }

        public int CustomerCount()
        {
            return _customers.Count;
        }
    }
}

File: tests/fixtures/sample_code/Sample.java
package com.example.service;

import java.util.List;
import java.util.ArrayList;

/**
 * Sample Java service class for testing.
 */
public class OrderService {
    private final List<String> orders;

    public OrderService() {
        this.orders = new ArrayList<>();
    }

    public void addOrder(String orderId) {
        if (orderId != null && !orderId.isEmpty()) {
            this.orders.add(orderId);
        }
    }

    public int getOrderCount() {
        return this.orders.size();
    }
}
